ui/app/components/app/dropdowns/components/menu.js

```diff
@@ -1,61 +1,81 @@
-const inherits = require('util').inherits
-const Component = require('react').Component
-const h = require('react-hyperscript')
+import PropTypes from 'prop-types'
+import React from 'react'
+import classnames from 'classnames'
 
-inherits(Menu, Component)
-function Menu () {
-  Component.call(this)
+function Menu (props) {
+  const { className, children, isShowing } = props
+  return isShowing
+    ? <div className={classnames('menu', className)}>{children}</div>
+    : <noscript />
 }
 
-Menu.prototype.render = function () {
-  const { className = '', children, isShowing } = this.props
-  return isShowing
-    ? h('div', { className: `menu ${className}` }, children)
-    : h('noscript')
+Menu.defaultProps = {
+  className: '',
+  isShowing: false,
+  children: null,
 }
 
-inherits(Item, Component)
-function Item () {
-  Component.call(this)
+Menu.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
+  isShowing: PropTypes.bool,
 }
 
-Item.prototype.render = function () {
+function Item (props) {
   const {
     icon,
     children,
     text,
     subText,
-    className = '',
+    className,
     onClick,
-  } = this.props
+  } = props
+
   const itemClassName = `menu__item ${className} ${onClick ? 'menu__item--clickable' : ''}`
```

Comment:
Probably better expressed with `classnames`

Reply:
Yeah I can update it